src/tests/test_rng.cpp

```diff
@@ -771,6 +771,21 @@ class System_RNG_Tests final : public Test
             rng.add_entropy(out_buf.data(), out_buf.size());
             }
 
+#if (BOTAN_MP_WORD_BITS == 64)
```

Comment:
Use `if constexpr(sizeof(size_t) > 4)` for this

Reply:
I did have this originally but reverted it as I'm still on 2.x but I see 3.0 supports c++17 so will change.